src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -7267,24 +7334,45 @@ String StorageReplicatedMergeTree::getSharedDataReplica(
     return best_replica;
 }
 
+
+Strings StorageReplicatedMergeTree::getZeroCopyPartPath(const MergeTreeSettings & settings, DiskType disk_type, const String & table_uuid,
+    const String & part_name, const String & zookeeper_path_old)
+{
+    Strings res;
+
+    String zero_copy = fmt::format("zero_copy_{}", toString(disk_type));
+
+    String new_path = fs::path(settings.remote_fs_zero_copy_zookeeper_path.toString()) / zero_copy / table_uuid / part_name;
```

Comment:
So now we store all zero copy data on a single path in ZK?

Reply:
It splitted by table_id.
I can't use something like table zookeeper_path, because in this case after drop table and recreate with same zookeeper path will be collisions between old and new part locks.